clients/java/src/main/java/io/camunda/client/CamundaClient.java

```diff
@@ -585,4 +586,20 @@ static CamundaClientCloudBuilderStep1 newCloudClientBuilder() {
    * @return a builder for the process instance query
    */
   ProcessInstanceQuery newProcessInstanceQuery();
+
+  /**
+   * Executes a search request to query user tasks.
+   *
+   * <pre>
+   * zeebeClient
+   *  .newUserTaskInstanceQuery()
+   *  .filter((f) -> f.userTaskKey(userTaskKey))
+   *  .sort((s) -> s.startDate().asc())
+   *  .page((p) -> p.limit(100))
+   *  .send();
+   * </pre>
+   *
+   * @return a builder for the user task query
+   */
+  UserTaskQuery newUserTaskInstanceQuery();
```

Comment:
🔧 Name-wise, does `newUserTaskQuery` suffice? There is no such term as "userTaskInstance" anywhere else, as far as I know.

Reply:
ups, this was a copy/paste mistake